Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -192,10 +192,6 @@
     <string name="risk_card_outdated_risk_body">"Your exposure logging could not be updated for more than 24 hours."</string>
     <!-- XTXT: risk card - outdated risk manual, calculation couldn't be updated in the last 48 hours -->
     <string name="risk_card_outdated_manual_risk_body">"Your risk status has not been updated for more than 48 hours. Please update your risk status."</string>
-    <!-- XHED: risk card - loading headline, update is ongoing -->
```

Comment:
please only remove strings in the german and the default string file. Treat the other files as auto generated, this can lead to unwanted merge conflicts.

Reply:
my fault: only add german strings, also the default strings will be overwritten by an update.

Reply:
I tried, but the CircleCI checks wouldn't pass then.